site/content/en/docs/tasks/run/kubeflow/trainjobs.md

```diff
@@ -0,0 +1,261 @@
+---
+title: "Run a TrainJob"
+date: 2024-11-05
+weight: 7
+description: >
+  Run a Kueue scheduled TrainJob from Kubeflow Trainer v2
```

Comment:
Seems like some rendering issue to solve:

<img width="1489" height="734" alt="Image" src="https://github.com/user-attachments/assets/2df3f552-720b-44fc-9898-4e936e344906" />